### PR TITLE
gh-94808: Coverage: Test that maximum indentation level is handled

### PR DESCRIPTION
**Pull request by mdboom**
* Issue: gh-94808

### CODE REVIEW

**Comment by iritkatriel** [comment]
What is the source file/function that this test covering?

**Comment by mdboom** [comment]
In `tokenizer.c`, in [tok_get](http://droettboom.com/cpython-coverage/llvm-coverage/coverage/home/mdboom/Work/builds/cpython/Parser/tokenizer.c.html#L1426)

File: Lib/test/test_tokenize.py [hunk 1]
```diff
@@ -2512,6 +2512,24 @@ def get_tokens(string):
...
+
+        def generate_source(indents):
+            source = ''.join(('  ' * x) + 'if True:\n' for x in range(indents))
+            source += '  ' * indents + 'pass\n'
+            return source
+
+        valid = generate_source(MAXINDENT - 1)
+        tokens = list(_generate_tokens_from_c_tokenizer(valid))
+        self.assertEqual(tokens[-1].type, DEDENT)
+        exec(valid)
```
Comment (serhiy-storchaka) [inline Lib/test/test_tokenize.py]: Would not `compile()` be enough?

**Review by serhiy-storchaka — APPROVED** [review]

File: Lib/test/test_tokenize.py (hunk 1 above)
Comment (mdboom) [reply Lib/test/test_tokenize.py]: Good idea.  Updated.

**Review by nanjekyejoannah — APPROVED** [review]
Thanks @mdboom , seems you handled the requested review.

**Pull request merged**